apps/trading/lib/hooks/use-market-click-handler.ts

```diff
@@ -20,20 +20,8 @@ export const useMarketClickHandler = (replace = false) => {
   );
 };
 
-export const useMarketLiquidityClickHandler = (replace = false) => {
-  const navigate = useNavigate();
-  const { marketId } = useParams();
-  const { pathname } = useLocation();
-  const isLiquidityPage = pathname.match(/^\/liquidity\/(.+)/);
-  return useCallback(
-    (selectedId: string, metaKey?: boolean) => {
-      const link = Links[Routes.LIQUIDITY](selectedId);
-      if (metaKey) {
-        window.open(`/#${link}`, '_blank');
-      } else if (selectedId !== marketId || !isLiquidityPage) {
-        navigate(link, { replace });
-      }
-    },
-    [navigate, marketId, replace, isLiquidityPage]
-  );
+export const useMarketLiquidityClickHandler = () => {
```

Comment:
Why this change here?

Reply:
I think we probably only want _blank if metaKey is true

Reply:
updated to keep the metakey. 
the method was changed because it was not working correctly. navigation on LP view was broken
even in mainnet